enhancements/hypershift/hostedClusterInfrastructure.md

```diff
@@ -0,0 +1,166 @@
+---
+title: hostedClusterInfrastructure
+
+authors:
+  - "@jnpacker"
+  
+reviewers:
+  - "@csrwng"
+  - "@enxebre"
+  - "@sjenning"
+  - "@alvaroaleman"
+  - "@derekwaynecarr"
+  - "@imain"
+  - "@yuqi-zhang"
+  
+approvers:
+  - "@csrwng"
+  - "@enxebre"
+  - "@sjenning"
+  - "@alvaroaleman"
+  - "@derekwaynecarr"
+  - "@imain"
+
+api-approvers:
+  - "@csrwng"
+  - "@enxebre"
+  - "@sjenning"
+  - "@alvaroaleman"
+  - "@derekwaynecarr"
+  - "@imain"
+
+tracking-link:
+  - https://issues.redhat.com/browse/ACM-1554
+
+creation-date: 2022-08-08
+last-updated: 2022-08-08
+---
+
+# Hosted Cluster Infrastructure
+
+## Summary
+This enhnacement was born out of the Untangling HypershiftDeployment, HostedCluster and NodePool document (see glossary). With the deprecation (removal) of HypershiftDeployment for GA, ACM/MCE requires a means to track and manage Infra for Cloud Providers for its primary user Cluster Administrator (multi-service cluster administrator).
+HostedClusterInfrastructure would be used to provision or track Cloud Provider infrastructure objects that can be used when provisioning HostedClusters using those providers.
+
+## Glossary
+- Untangling HypershiftDeployment, HostedCluster and NodePool Google document - [Google Document - Option D](https://docs.google.com/document/d/1VabhUQa_uZlWO6mf9UHS5lGtnH_X915cih4ARJbNibc/edit#heading=h.iuee7tji8zz4)
+
+## Motivation
+ACM and MCE primary Cloud Provider use cases already provision these resources and manage these resources, this provides feature parity with Standalone flows that ACM Cluster Administrator's are used to.  This also allows to additional opporunities:
+  1. Infrastructure can be pre-created using an ACM Cloud Provider credential or by pre-populating the Custom Resource with values and applying it to the ACM/MCE hub. These can then be consumed by the ACM/MCE UI or CLI, and is a nice option, as the AWS credential does not need to be exposed to the user creating the HostedCluster.
+  2. You can re-use the HostedClusterInfrastructure resource between HostedClusters. This is especially useful in Azure, where it takes longer to create infrastructure then the hosted control plane.
+
+### User Stories
+- As a Cluster Administrator I want to create one or more HostedClusterInfrastructure resources for my team to consume.
```

Comment:
I'd say this is a cluster service provider / infra admin, for a cluster admin we refer to the persona with full rbac in a guest cluster https://hypershift-docs.netlify.app/reference/concepts-and-personas/

Reply:
I do think it ends up being a little confusing as Cluster Administrator is used earlier in the doc with MCE/ACM, but in our case that persona is the manager of multiple clusters (guess that definition will should be adjusted)

Reply:
I tried to make it more clear by including ACM/MCE and a bracket about infrastructure administrator.